### DISCUSSION TITLE
Allow Multishot and Piercing Together

### DISCUSSION THREAD

**Discussion by DevReaper0** [PurpurMC/Purpur#1066] [Feature Requests]
A configuration option similar to 'allow-infinity-and-mending-together' but for multishot and piercing.

**Comment by BillyGalbreath** [comment]
The only reason we have the `allow-infinity-and-mending-together` option is because that [_used_ to be a regular mechanic](https://minecraft.fandom.com/wiki/Java_Edition_1.11.1#Gameplay_2) in the game prior to 1.11.1.
